### PR TITLE
fix for removed View.propTypes

### PR DESCRIPTION
**Pull request by yusukeshibata**
Fixed the issue: #279

### CODE REVIEW

**Comment by holden-caulfield** [comment]
Please merge this... so far the library can't be use on a production release that uses the latest RN because of this bug

**Comment by scottsgcho** [comment]
Could you merge this already?

**Comment by brunoczo** [comment]
I've wasted hours looking for the problem. Please fix this.  To solve local, i edit the node_module.

**Comment by flippinjoe** [comment]
Sweet Jesus.  Push the button

**Comment by mattiamanzati** [comment]
+1

File: index.ios.js [hunk 1]
```diff
@@ -90,7 +91,7 @@ class FBLogin extends Component {
 }
 
 FBLogin.propTypes = {
-  style: View.propTypes.style,
+  style: ViewPropTypes.style,
```
Comment (habovh) [inline index.ios.js]: May I suggest to use a ternary here to keep the library backwards compatible (< RN 0.49)?

Like this:
```
const viewPropTypes = ViewPropTypes || View.propTypes
FBLogin.propTypes = {
    [...],
    style: viewPropTypes.style,
    [...],
}
```

**Comment by habovh** [comment]
@yusukeshibata it seems you also forgot to update their example component at `example/components/facebook/FBLoginMock.js`, but that's not preventing the component itself from being used.

**Comment by kayzenkayzen** [comment]
Please merge!!

**Comment by nachocifu** [comment]
+1

**Comment by yusukeshibata** [comment]
Thanks @habovh , I pushed new commits.

**Comment by Dalamar** [comment]
Please make it finally available in the master branch.
I can't release my app.
Worth to mention that it took me 2 days to figure out why does my app crashes on released versions.

**Comment by habovh** [comment]
@Dalamar If you're in a hurry you can still use @yusukeshibata's fork from GitHub instead of the official npm package until it gets merged.

**Comment by magus** [comment]
Thanks for reporting this, the fix is out in version 1.6.1!

**Pull request merged**